frontend/apps/reader/modules/readertoc.lua

```diff
@@ -702,6 +715,8 @@ function ReaderToc:onShowToc()
         icon_height = icon_size,
         bordersize = 0,
         show_parent = self,
+        callback = function(index) self:expandToc(index) end,
+        onTapSelectButton = function() end, -- pass thirugh taps to onMenuSelect
```

Comment:
thirugh !
(Google says this typo has been made only 2 240 times in the history of the world, congrats !)

Reply:
done